index.js

```diff
@@ -1,7 +1,5 @@
 'use strict';
-var URL = 'https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.json'
-var got = require('got');
 module.exports = function (cb) {
-	if (typeof cb !== 'function') throw new Error("Missing callback");
-	got(URL,cb)
+  if (typeof cb !== 'function') throw new Error("Missing callback");
+  require('fs').readFile(__dirname + '/resources/blns.json', cb);
```

Comment:
We don't need callback anymore.

See below as reference:

http://stackoverflow.com/questions/7163061/is-there-a-require-for-json-in-node-js

Just bump to v2.0, for this breaking change.